icaruscode/CRT/CRTDecoder/Macros/calib.C

```diff
@@ -0,0 +1,265 @@
+  calTree->Branch("gainPed",      fGainPed,      "gainPed[32]/F");
+  calTree->Branch("gainPedErr",   fGainPedErr,   "gainPedErr[32]/F");
+  calTree->Branch("nPeak",        fNpeak,        "nPeak[32]/s");
+  calTree->Branch("peakXsqr",     fPeakXsqr,     "peakXsqr[32][5]/F");
+  calTree->Branch("peakNdf",      fPeakNdf,      "peakNdf[32][5]/s");
+  calTree->Branch("peakMean",     fPeakMean,     "peakMean[32][5]/F");
+  calTree->Branch("peakMeanErr",  fPeakMeanErr,  "peakMeanErr[32][5]/F");
+  calTree->Branch("peakNorm",     fPeakNorm,     "peakNorm[32][5]/F");
+  calTree->Branch("peakNormErr",  fPeakNormErr,  "peakNormErr[32][5]/F");
+  calTree->Branch("peakSigma",    fPeakSigma,    "peakSigma[32][5]/F");
+  calTree->Branch("peakSigmaErr", fPeakSigmaErr, "peakSigmaErr[32][5]/F");
+  calTree->Branch("ped",          fPed,          "ped[32]/F");
+  calTree->Branch("pedErr",       fPedErr,       "pedErr[32]/F");
+  calTree->Branch("pedXsqr",      fPedXsqr,      "pedXsqr[32]/F");
+  calTree->Branch("pedNdf",       fPedNdf,       "pedNdf[32]/s");
+  calTree->Branch("pedSigma",     fPedSigma,     "pedSigma[32]/F");
+  calTree->Branch("pedSigmaErr",  fPedSigmaErr,  "pedSigmaErr[32]/F");
+  calTree->Branch("pedNorm",      fPedNorm,      "pedNorm[32]/F");
+  calTree->Branch("pedNormErr",   fPedNormErr,   "pedNormErr[32]/F");
+  calTree->Branch("threshAdc",    fThreshADC,    "threshAdc[32]/I");
+  calTree->Branch("threshPe",     fThreshPE,     "threshPe[32]/F");
+  calTree->Branch("nAbove",       fNabove,       "nAbove[32]/I");
+  calTree->Branch("stats",	  fChanStats,	 "stats[32]/L");
+
+/*  std::vector<int> macadd = {1,3,6,7,
+                             4,5,8,9,
+                             10,11,13,14,15,16,    // west-center outer
+                             17,19,21,22,23,24,    // west-center inner
+                             25,26,27,28,29,30,    // east-center outer
+                             31,32,33,34,35,36,    // east-center inner
+                             37,38,39,40,41,42,    // east-north outer
+                             43,44,45,46,47,48,    // east-north inner
+                             49,50,51,52,53,54,    // east-south outer
+                             55,56,57,58,59,60,    // east-south inner
+                             61,62,63,64,65,66,    // west-north outer
+                             67,68,69,70,71,72,    // west-north inner
+                             73,74,75,76,77,78,    // west-south outer
+                             79,80,81,82,83,84,    // west-south inner
+			     85,86,87,88,89,90,91, // south outer
+			     92,93,94,95,96,97};   // south inner
+*/
+std::vector<int> macadd = {1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93};
```

Comment:
Not a show stopper, but in general I'd suggest to read constants like this from a fhicl file.